docs/flow/modin/base.rst

```diff
@@ -0,0 +1,12 @@
+BasePandasDataset
```

Comment:
Should this be `modin/pandas/base.rst` instead?

Reply:
Ok.